src/Insurello.RabbitMqClient/MqClient.fs

```diff
@@ -197,13 +208,26 @@ module MqClient =
                 ()
 
     let private closeRpcConsumer: Model -> unit =
-        fun (Model model) -> if model.rpcConsumer.Model.IsOpen then model.rpcConsumer.Model.Close() else ()
+        fun (Model model) ->
+            if model.rpcConsumer.Model.IsOpen then
+                model.rpcConsumer.Model.Close()
+            else
+                ()
 
     let private closeChannelConsumer: Model -> unit =
-        fun (Model model) -> if model.channelConsumer.Model.IsOpen then model.channelConsumer.Model.Close() else ()
+        fun (Model model) ->
+            if model.channelConsumer.Model.IsOpen then
+                model.channelConsumer.Model.Close()
+            else
+                ()
 
     let close: Model -> unit =
```

Comment:
We can add a fancy `/// summary` comment

Reply:
> We can add a fancy `/// summary` comment, similar to the comment below but changed tense.

** similar to the comment below but changed tense.**

Reply:
Put yourself in the user of this api and write what you would have wanted to read.